lib/mayaUsd/ufe/StagesSubject.cpp

```diff
@@ -69,7 +70,26 @@ bool inAttributeChangedNotificationGuard()
     return attributeChangedNotificationGuardCount.load() > 0;
 }
 
-std::unordered_map<Ufe::Path, std::string> pendingAttributeChangedNotifications;
+std::unordered_multimap<Ufe::Path, TfToken> pendingAttributeChangedNotifications;
+
+void sendValueChanged(const Ufe::Path& ufePath, const TfToken& changedToken)
+{
+    Ufe::AttributeValueChanged vc(ufePath, changedToken.GetString());
+    Ufe::Attributes::notify(vc);
+
+    if (MAYAUSD_NS_DEF::ufe::UsdCamera::isCameraToken(changedToken)) {
```

Comment:
The macro MAYAUSD_NS_DEF is only meant to be used when declaring a namespace. Here you should just use "MayaUsd::"